### PR TITLE
add options for hat and christmas outfit for custom tromboners

### PR DESCRIPTION
**Pull request by DerpyChap**
i thought about not adding the hat option considering it's unused and could potentially get removed, but there are so many other unused assets in this game that i think that'd be unlikely. also, the game seems to be set up with multiple hats in mind internally, so if anything i think it's possible this feature could return in some expanded form.

i apparently also messed up the Editor config in the solution so i fixed that quickly.

### CODE REVIEW

File: CustomTracks/Backgrounds/CustomBackground.cs [hunk 1]
```diff
@@ -166,6 +166,25 @@ public override void SetUpBackground(BGController controller, GameObject bg)
...
+                materials[0] = tromboner.controller.costume_alt;
+                tromboner.controller.bodymesh.materials = materials;
+            }
+            
+            var chosen_hat = trombonePlaceholder.TromboneHat == TromboneHat.DoNotOverride
+            ? GlobalVariables.chosen_hat : (int)trombonePlaceholder.TromboneHat;
+
+            if (chosen_hat > 0)
+            {
+                GameObject hat = Object.Instantiate(controller.gamecontroller.hats[chosen_hat - 1],
```
Comment (hedgehog1029) [inline CustomTracks/Backgrounds/CustomBackground.cs]: we have `GameController` bound to a local variable `gameController` in this scope

File: CustomTracks/Backgrounds/CustomBackground.cs [hunk 2]
```diff
@@ -182,6 +201,8 @@ public override void SetUpBackground(BGController controller, GameObject bg)
                     tromboner.controller.p_tube.transform.localScale = new Vector3(1f, 1f, 2.25f);
                     break;
             }
+
+
```
Comment (hedgehog1029) [inline CustomTracks/Backgrounds/CustomBackground.cs]: random spacing go

**Review by hedgehog1029 — CHANGES_REQUESTED** [review]
looks good just some minor bits

File: CustomTracks/Backgrounds/CustomBackground.cs (hunk 1 above)
Comment (DerpyChap) [reply CustomTracks/Backgrounds/CustomBackground.cs]: sorted 👍

File: CustomTracks/Backgrounds/CustomBackground.cs (hunk 2 above)
Comment (DerpyChap) [reply CustomTracks/Backgrounds/CustomBackground.cs]: not sure how that got there 👀

**Review by hedgehog1029 — APPROVED** [review]

**Pull request merged**